app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML preview, math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose"
+LICENSE="GNU GPL v2"
```

Comment:
This package contains code [licensed](https://github.com/cloose/CuteMarkEd/blob/develop/LICENSE.md) under three different terms

```diff
-LICENSE="GNU GPL v2"
+LICENSE="GNU GPL v2
+	BSD (3-clause)
+	MIT"

Reply:
Why BSD? Archive doesn't contain Discount code.

Reply:
If it doesn't then feel free to drop BSD.